### ISSUE TITLE
Syndie lantern light appears in inventory area when held, not body

### ISSUE THREAD

**Issue by Addust** [tgstation/tgstation#68912]
Reporting client version: 514.1585

## Round ID:
[187692](https://scrubby.melonmesa.com/round/187692)

## Testmerges:
- [Adds the Mothroach](https://www.github.com/tgstation/tgstation/pull/68763)
- [Refactors SM Delaminations to be strategies](https://www.github.com/tgstation/tgstation/pull/68599)
- [Lemon does pain](https://www.github.com/tgstation/tgstation/pull/67902)

## Reproduction:
1. go to listening post
2. pick up the moff lamp in the lobby
3. turn on
4. light at bottom of screen
5. scream shitcode out and file a bug report

**Comment by Mooshimi** [comment]
Please post this in https://github.com/tgstation/tgstation/pull/67902

**Comment by LemonInTheDark** [comment]
![image](https://user-images.githubusercontent.com/58055496/182788317-dbedaebb-ca3c-4158-8ac9-6862007f4f0f.png)
Does not replicate, do you have more details?

**Comment by Addust** [comment]
Tested in the Syndicate Listening Post on Terry.

**Comment by Addust** [comment]
May be an issue with the specific mofflamp, I ain't sure

**Comment by msgerbs** [comment]
Oh, I fixed this. Didn't realize there was a duplicate report. Fixed in #69754

**Comment by msgerbs** [comment]
TL;DR Yes it was that specific lamp. Bad mapper, bad!
